packages/stepper/src/Step.tsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+
+const Step = React.Fragment;
+export default Step;
```

Comment:
If we have a `Step` component, we should consider doing a bit more with it (see `Select/Option`)

Reply:
I think the `Step` component isn't really needed - below is what we used to have:
```
export const Step = function Step({
  children,
}: {
  children: StepContents;
}): StepElement {
  return <>{children}</>;
};
```
I've been told though that applications are using the `Step` component in their apps, so just having something exported to pass children through makes sure their apps don't break. 

I've thought about things to add, but because every `Step` label content is styled through children being passed directly through, devs have full flexibility over their styles. Customization of styles on other parts of the component should probably be limited.

Reply:
I think it's needed to make it clear that we're adding elements & styles to the markup that a user provides. It's not immediately intuitive what something like this would do 
```js
<Stepper>
  <H1>Some step</H1>
  <H2>Some other step</H2>
</Stepper>
```

but this is a bit clearer that what I passed in is going to be rendered as a step
```js
<Stepper>
  <Step>
    <H1>Some step</H1>
  </Step>
  <Step>
    <H2>Some other step</H2>
  <Step>
</Stepper>
```

Reply:
On the other side, it doesn't make much sense to use a standalone `<Step/>`, so we should add the restrictions like we have in the [Option component](https://github.com/mongodb/leafygreen-ui/blob/main/packages/select/src/Option.tsx#L302-L304)